x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -427,7 +426,7 @@ const prepareDefinitionSectionListItems = (
   rule: Partial<RuleResponse>,
   isInteractive: boolean,
   savedQuery: SavedQuery | undefined,
-  { alertSuppressionForIndicatorMatchRuleEnabled }: Partial<ExperimentalFeatures> = {}
+  isSuppressionEnabled: boolean
```

Comment:
nit: maybe having the fourth argument as object `params` would be better? Can be easily extend in future with new props

Reply:
The utilization of the fourth argument as an object was appropriate in the context of ExperimentalFeatures because it encompassed multiple pertinent properties. Given that this method accepts all the properties individually, exclusively isolating the properties beginning with isSuppressionEnabled could lead to confusion